### DISCUSSION TITLE
District 12 Meetings Not All Showing up in JSON

### DISCUSSION THREAD

**Discussion by joshreisner** [code4recovery/12-step-meeting-list#1561] [Q&A] [answered]
there are 112 meetings here https://docs.google.com/spreadsheets/d/1cERocR4UEey_Hoq_pJkDI6XSLm2b-C0v6XyiSEiGWKQ/edit?gid=687617754#gid=687617754

but only 51 meetings are in: https://sheets.code4recovery.org/storage/1cERocR4UEey_Hoq_pJkDI6XSLm2b-C0v6XyiSEiGWKQ.json

**Comment by westlinksonline** [comment, marked as answer]
There are only 52 rows with data in column C, Name.

**Reply by westlinksonline** [reply]
I haven't tested this, but assume it is a likely cause of the issue.

**Reply by joshreisner** [reply]
NICE that's the problem. meeting name and id are always required. i see the sheets service is not complaining about the missing names, so i've [created an issue](https://github.com/code4recovery/sheets/issues/33) to start showing an error on those listings
